examples/datalog/merge_intervals.flix

```diff
@@ -0,0 +1,21 @@
+// https://leetcode.com/problems/merge-intervals/
+// Given an array of intervals where intervals[i] = [start_i, end_i], merge all overlapping intervals, and
```

Comment:
Since the specification asks for a function accepting an array of intervals and returning an array of intervals, perhaps we should write a function that does that (rather than printing), and then call it from main?

Alternatively, we can change the description.

Reply:
Yup I can write a function that does that.